modules/va_forms/app/sidekiq/va_forms/form_builder.rb

```diff
@@ -22,8 +21,8 @@ class FormBuilder
       if error.message == FORM_FETCH_ERROR_MESSAGE
         form = VAForms::Form.find_by(row_id:)
         url = VAForms::Form.normalized_form_url(args.dig('fieldVaFormUrl', 'uri'))
-        form_previously_valid = form&.valid_pdf
-        form&.update!(valid_pdf: false, sha256: nil, url:)
+        form_previously_valid = form.valid_pdf
+        form.update!(valid_pdf: false, sha256: nil, url:)
```

Comment:
Safe reference isn't necessary here because this branch of the code is only reached if the form was successfully saved previously.